docs/schema.graphql

```diff
@@ -742,6 +743,47 @@ type RootQueryType {
       }
     }
   }
+  
+  bridge-account-udt-example:
+  query {
+    account(
+      input: {
+        script_hash: "0x3E1301E759261B676CE68D0D97936CD431A4AF2A34072AA94E44655909765EB4"
+      }
+    ) {
+      udt {
+        id
+        name
+        bridge_account_id
+        type
+      }
+      bridged_udt {
+        id
+        name
+        bridge_account_id
+        type
+      }
+    }
+  }
+  
+  {
+    "data": {
+      "account": {
+        "bridged_udt": {
+          "bridge_account_id": 36050,
```

Comment:
It seems to be the `native udt id`. And why is `bridge account id` a number while `id` is a string

Reply:
I misunderstood the updated field